src/appleseed/renderer/modeling/bssrdf/betterdipolebssrdf.cpp

```diff
@@ -87,13 +87,20 @@ namespace
             return Model;
         }
 
-        virtual void prepare_inputs(void* data) const APPLESEED_OVERRIDE
+        virtual void prepare_inputs(
+            const ShadingPoint& shading_point,
```

Comment:
Wondering if this shouldn't be called `outgoing_point` to make it clear which point it is...